DoxyfileC

```diff
@@ -0,0 +1,2830 @@
+# Doxyfile 1.9.8
```

Comment:
i think we should generate the file with `1.11.0` version because we use it

Reply:
it's adapted copy DoxygenCpp. So we need to update all this files, but i suggest doing it in other PR